test_polymorphic_value.cpp

```diff
@@ -28,9 +28,9 @@ using namespace stdx;
 #endif
 
 static_assert(sizeof(polymorphic_value<SmallBase>) == 64 + sizeof(void*));
-static_assert(sizeof(polymorphic_value < SmallBase, { .size = 2 } > ) == 2 * sizeof(void*));
+static_assert(sizeof(polymorphic_value < SmallBase, polymorphic_value_options{ .size = 2 } > ) == 2 * sizeof(void*));
```

Comment:
I'm pretty sure the original could should work in C++20, which compiler needs the class name? If the standard needs the class name maybe this idea of using a policy struct should be abandoned.

Reply:
I'm using clang. I'm not a fan of the policy struct in general but would like to see you branch build and run locally and under CI. We have some more PRs to put forward but this one is needed first.